test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Cast/CastTest.cs

```diff
@@ -49,30 +49,35 @@ protected override void UpdateConfiguration(WebRouteConfiguration configuration)
                 {
                     // To Edm.String
                     combinations.Add(dataSourceType, "?$filter=cast('Name1',Edm.String) eq Name", 1);
-                    combinations.Add(dataSourceType, "?$filter=contains(cast(Name,Edm.String),'Name')", 3);
-                    combinations.Add(dataSourceType, "?$filter=cast(Microsoft.Test.E2E.AspNet.OData.Cast.Domain'Civil',Edm.String) eq '2'", 3);
-                    combinations.Add(dataSourceType, "?$filter=cast(Domain,Edm.String) eq '3'", 1);
-                    combinations.Add(dataSourceType, "?$filter=cast(ID,Edm.String) gt '1'", 2);
+                    combinations.Add(dataSourceType, "?$filter=contains(cast(Name,Edm.String),'Name')", 6);
```

Comment:
Why changing existing test case? Is your change breaking change? Can you point me to where this is used?

Reply:
Yes added entities